### DISCUSSION TITLE
Rebuilding hammer after 1.0 or 2.0 release

### DISCUSSION THREAD

**Discussion by Gravy59** [StrataSource/Portal-2-Community-Edition#645] [Ideas]
I was on _something_ when I wrote this

**Comment by vrad-exe** [comment]
Hammer's UI is now in the process of being converted to QT which is a much more modern and non-Windows-specific UI framework. Not sure if we're going to completely rewrite Hammer since it'd be a lot of work.

**Comment by AWildErin** [comment]
The focus will primarily be porting the UI from MFC to Qt. Later on down the line, we want to make some more improvements to Hammer, which may include editing parts of it including the UI. But as of now, the focus is just porting the UI
